src/main/java/uk/gov/justice/log/search/SearchCriteria.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.justice.log.search;
+
+import uk.gov.justice.log.utils.CommonConstant;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SearchCriteria {
+    private int durationMinutes;
+    private List<String> keywords;
+    private List<String> regexes = new ArrayList<>();
```

Comment:
Why create a default list here for this field but not the others? Best done in the constructor, I think.